deepfence_frontend/apps/dashboard/src/utils/date.ts

```diff
@@ -11,3 +11,7 @@ export const formatMilliseconds = (date: number | string) => {
 export const formatToRelativeTimeFromNow = (date: string | Date | number) => {
   return dayjs(date).fromNow();
 };
+
+export const formatToString = (date: string | Date | number) => {
```

Comment:
use `formatMilliseconds`, we don't need this function. add date type to `formatMilliseconds` if needed

Reply:
updated formatMilliseconds to accept date